app/src/lib/stores/app-store.ts

```diff
@@ -2077,6 +2087,41 @@ export class AppStore extends TypedBaseStore<IAppState> {
     this.stashedFilesWidth = constrain(this.stashedFilesWidth, 100, filesMax)
   }
 
+  /**
+   * Calculate the constraints of the resizable pane in the pull request dialog
+   * whenever the window dimensions change.
+   */
+  private updatePullRequestResizableConstraints() {
+    // TODO: Get width of PR dialog -> determine if we will have default width
```

Comment:
At this point in time, the width of the dialog is static; thus, solving for this is not useful. I will address updating this logic when I solve the responsive width of the dialog challenge... if for some reason we deem having a responsive dialog unrealistic, this method likely isn't needed and we can just initialize the min/max constraints.